ui/src/application/Applications.tsx

```diff
@@ -152,7 +153,7 @@ const Row: SFC<IRowProps> = observer(
         <TableRow>
             <TableCell padding="checkbox">
                 <div style={{display: 'flex'}}>
-                    <Avatar src={image} />
+                    <Avatar src={config.get('url') + image} />
```

Comment:
The message.tsx also has a component which displays the image. This need to be changed to.